### PR TITLE
[release-3.10]Do not stop and delete openvswitch when NSX is used #8134 and #8015

### PR DESCRIPTION
**Pull request by yasensim**
PR fixing #8134 and #8015 as we agreed in PR #9895

### CODE REVIEW

**Comment by yasensim** [comment]
@vrutkovs it is done for release-3.10 as requested in PR #9895 !

File: roles/openshift_node/tasks/upgrade/config_changes.yml [hunk 1]
```diff
@@ -43,7 +43,9 @@
   package:
     name: openvswitch
     state: absent
-  when: not openshift_is_atomic | bool
```
[the remark this answers is not in the document]
Comment (yasensim) [reply roles/openshift_node/tasks/upgrade/config_changes.yml]: done!

File: roles/openshift_node/tasks/upgrade/config_changes.yml [hunk 2]
```diff
@@ -53,7 +53,13 @@
   - /etc/systemd/system/docker.service.d/docker-sdn-ovs.conf
   - /etc/systemd/system/atomic-openshift-node-dep.service
   - /etc/systemd/system/origin-node-dep.service
-  - /etc/systemd/system/openvswitch.service
+
+- name: Remove openvswitch service information
+  file:
+    path: /etc/systemd/system/openvswitch.service
+    state: absent
+  when:
```
Comment (vrutkovs) [inline roles/openshift_node/tasks/upgrade/config_changes.yml]:
```
  61:8      error    trailing spaces  (trailing-spaces)
  62:5      error    wrong indentation: expected 2 but found 4  (indentation)
```

Comment (yasensim) [reply roles/openshift_node/tasks/upgrade/config_changes.yml]: yeah sorry, I overslept this.

**Comment by yasensim** [comment]
@vrutkovs it must be good now. I plan to submit the same PR for release-3.11 as well.

**Comment by yasensim** [comment]
@vrutkovs it seems all tests have passed and only your approval is pending. 
Thanks,
Yasen

File: roles/openshift_node/tasks/upgrade/stop_services.yml [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
-  - openvswitch
   - "{{ openshift_service_type }}-node"
   failed_when: false
 
+- name: Stop openvswitch service
+  service:
+    name: openvswitch
+    state: stopped
+  when: not openshift_use_nsx
+  failed_when: false
```
Comment (vrutkovs) [inline roles/openshift_node/tasks/upgrade/stop_services.yml]: That's not good. Why would it fail here? Why is this line necessary?

Comment (yasensim) [reply roles/openshift_node/tasks/upgrade/stop_services.yml]: I didnt want to change this as it was in the original task. I will remove it.

Comment (vrutkovs) [reply roles/openshift_node/tasks/upgrade/stop_services.yml]: Lets keep it - afaik this is for Atomic where the service would have a different name and its safe to skip this

**Comment by yasensim** [comment]
@vrutkovs I will close this PR and send a new clean one
